code/datums/atmosphere/planetary.dm

```diff
@@ -0,0 +1,26 @@
+// Atmos types used for planetary airs
+/datum/atmosphere/lavaland
+	id = LAVALAND_DEFAULT_ATMOS
+
+	base_gases = list(
+		/datum/gas/oxygen=5,
+		/datum/gas/nitrogen=10,
+	)
+	normal_gases = list(
+		/datum/gas/oxygen=10,
+		/datum/gas/nitrogen=10,
+		/datum/gas/carbon_dioxide=10,
+	)
+	restricted_gases = list(
+		/datum/gas/bz=10,
+		/datum/gas/miasma=10,
```

Comment:
also i'd recommend we just not do this for now since miasma is 

1. very hard to come by so it's expensive in cargo

2. causes diseases which is extremely annoying.

3. has an ugly ass overlay

Reply:
or at least tone down the amount

Reply:
Not adding more of a gas because it's rare seems like a circular argument. I like the idea of lavaland giving you diseases some rounds and if we decide to include then the proper move is to reduce the value in cargo.

Why is NT buying miasma anyway

Reply:
why does lavaland have so much miasma on it, miasma is like crap that comes off of dead corpses

and yeah to be honest i have no idea, i think the reasoning was is miasma is rare since its hard to harvest. i think it has fusion power too.

i’ve never seen a planet with rotting flesh anyways especially not one with lava on it. just doesn’t seem like it should be so common.

diseases on lavaland just doesn’t sound fun at all to be perfectly honest.

Reply:
I don't see any particularly compelling reason why the planet couldn't have diseases and toxic gas in the atmosphere, it doesn't necessarily have to represent something only coming from rotting corpses.

Yeah it could be turned down a bit but I've liked so far seeing people deal with it.